src/ros_filter.cpp

```diff
@@ -991,20 +995,21 @@ namespace RobotLocalization
              "\ntransform_timeout is " << tfTimeout_.toSec() <<
              "\nfrequency is " << frequency_ <<
              "\nsensor_timeout is " << filter_.getSensorTimeout() <<
-             "\ntwo_d_mode is " << (twoDMode_ ? "true" : "false") <<
-             "\nsmooth_lagged_data is " << (smoothLaggedData_ ? "true" : "false") <<
+             "\ntwo_d_mode is " << std::boolalpha << twoDMode_ <<
```

Comment:
Sorry, I printed out the new param here using `boolalpha`, and thought I might as well pick up the others.